infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -228,4 +229,12 @@ public void eth1DepositChainIdMismatch(int expectedChainId, int eth1ChainId) {
         expectedChainId,
         eth1ChainId);
   }
+
+  public void externalSignerStatus(final URL externalSignerUrl, boolean isReachable) {
+    if (isReachable) {
+      log.info("External signer is reachable at {}", externalSignerUrl);
+    } else {
+      log.warn("External signer is currently not reachable at {}", externalSignerUrl);
```

Comment:
nit: I'd make this an error and it's probably worth making it red with `ColorConsolePrinter.print`.

Reply:
fixed by upcheck PR